### ISSUE TITLE
fail to run?

### ISSUE THREAD

**Issue by cryoguy** [nobiot/org-transclusion#33]
I am getting the following error when I press <f12> to activate. any thoughts?  Also does this work with the whole org file too, or just the headlines?

```
if: Symbol’s value as variable is void: window-selection-change-functions
```

I am using Emacs 26.3 and Arch Linux newest version.

**Comment by nobiot** [comment]
Org-transclusion relies on the function in the error at the moment, and it is available with 27.1. You would need to upgrade Emacs.

Org-transclusion works for the whole org file, too. You can configure if you want to include the section before the first headline (default is not to include it).

**Comment by cryoguy** [comment]
I have resisted the 27.1 for a while.  I will do it for this interesting mode. Thanks!
